tests/parser.rs

```diff
@@ -19,10 +19,10 @@ use joker::track::Untrack;
 use serde_json::value::Value;
 use std::ffi::OsStr;
 use std::fs::{File, read_dir};
-use std::io::Read;
+use std::io::{Read, Write, stdout};
 use std::path::Path;
 use std::{thread, env};
-use test::{TestDesc, TestDescAndFn, TestName, TestFn, test_main};
+use test::{TestDesc, TestDescAndFn, TestName, TestFn, Bencher, TDynBenchFn, test_main};
```

Comment:
It's actually documented as part of Rust at https://doc.rust-lang.org/1.1.0/test/index.html - `rustc-test` just extracted it out of nightly build of Rust if I understand correctly from their commit descriptions. But yeah, I can do that.